database/src/main/resources/db/migration/V29__utbetaling_type.sql

```diff
@@ -0,0 +1,15 @@
+alter table utbetaling
+    add column if not exists
+        type text
+            default null;
+
+with linje as(
+	select l.id, l.beløp, l.utbetalingId, (select beløp from utbetalingslinje forrige where forrige.id = l.forrigeUtbetalingslinjeId) as forrigeBeløp from utbetalingslinje l
+) 
+update utbetaling 
+set type = (case
+	when linje.beløp = 0 then 'STANS'
+	when linje.forrigeBeløp = 0 then 'GJENOPPTA'
```

Comment:
Vi har vel ingen garanti for at når forrige beløp er 0, så er det en GJENOPPTA? Det kan vel ha sniket seg inn noe i mellom her. Er kanskje ikke så stor sjans for det da og er vel og lett å sjekke.